ProjectLighthouse.Servers.Website/Pages/UsersPage.cshtml.cs

```diff
@@ -37,8 +37,9 @@ public async Task<IActionResult> OnGet([FromRoute] int pageNumber, [FromQuery] s
 
         if (this.PageNumber < 0 || this.PageNumber >= this.PageAmount) return this.Redirect($"/users/{Math.Clamp(this.PageNumber, 0, this.PageAmount - 1)}");
 
-        this.Users = await this.Database.Users.Where(u => u.PermissionLevel != PermissionLevel.Banned && u.Username.Contains(this.SearchValue))
-            .Where(u => u.ProfileVisibility == PrivacyType.All) // TODO: change check for when user is logged in
+        this.Users = await this.Database.Users
+            .Where(u => u.Username.Contains(this.SearchValue))
```

Comment:
The same thing about the chaining style, we should either reformat the entire project or keep using the old style. We shouldn't be mixing